### ISSUE TITLE
eimp not compiled but ejabberd tries to start it in loop

### ISSUE THREAD

**Issue by mremond** [processone/ejabberd#2224]
> What version of ejabberd are you using?

Master HEAD

> What operating system (version) are you using?

MacOS 10.13

> How did you install ejabberd (source, package, distribution)?

Github clone

> What did not work as expected? Are there error messages in the log? What
> was the unexpected behavior? What was the expected result?

ejabberd tries to start eimp and fails in loop:

```
2018-01-14 11:58:14 =ERROR REPORT====
External eimp process (pid=46562) has terminated unexpectedly, restarting in a few seconds
==> var/log/ejabberd/ejabberd.log <==
2018-01-14 11:58:14.896 [error] <0.174.0> External eimp process (pid=46562) has terminated unexpectedly, restarting in a few seconds
```

I was expecting ejabberd not to attempt to start it as I did build is with configure option `--disable-graphics`.

**Comment by mremond** [comment]
It seems that even when compiled with `--disable-graphics`, the default config file include `mod_vcard_xupdate` when doing `make install`.

**Comment by mremond** [comment]
For unknown reason, GRAPHICS macro seems uncorrectly applied. If I manually force code to keep only the NOOP start_eimp function, it works fine.

**Comment by zinid** [comment]
Should be fixed in 0f86559d83a37d97e1ce18aa346a67f7d0b391e2
